alignments/sweet-qudt-mapping.ttl

```diff
@@ -0,0 +1,390 @@
+.
+reprSciUnits:perMeterCubed
+  skos:exactMatch <http://qudt.org/vocab/unit/PER-M3> ;
+.
+reprSciUnits:perMeterSquared
+  skos:exactMatch <http://qudt.org/vocab/unit/PER-M2> ;
+.
+reprSciUnits:perMil
+  skos:exactMatch <http://qudt.org/vocab/unit/PPTH> ;
+.
+reprSciUnits:perMole
+  skos:exactMatch <http://qudt.org/vocab/unit/PER-MOL> ;
+.
+reprSciUnits:perPascal
+  skos:exactMatch <http://qudt.org/vocab/unit/PER-PA> ;
+.
+reprSciUnits:perSecond
+  skos:exactMatch <http://qudt.org/vocab/unit/PER-SEC> ;
+.
+reprSciUnits:percent
+  skos:exactMatch <http://qudt.org/vocab/unit/PERCENT> ;
+.
+reprSciUnits:permil
+  skos:exactMatch <http://qudt.org/vocab/unit/PPTH> ;
+.
+reprSciUnits:peta
+  skos:exactMatch <http://qudt.org/2.1/vocab/prefix/Peta> ;
+.
+reprSciUnits:ppb
+  skos:exactMatch <http://qudt.org/vocab/unit/PPB> ;
+.
+reprSciUnits:ppm
+  skos:exactMatch <http://qudt.org/vocab/unit/PPM> ;
+.
+reprSciUnits:ppt
+  skos:exactMatch <http://qudt.org/vocab/unit/PPTR> ;
+.
+reprSciUnits:radian
+  skos:exactMatch <http://qudt.org/vocab/unit/RAD> ;
+.
+reprSciUnits:radianPerSecond
+  skos:exactMatch <http://qudt.org/vocab/unit/RAD-PER-SEC> ;
+.
+reprSciUnits:radianPerSecondSquared
+  skos:exactMatch <http://qudt.org/vocab/unit/RAD-PER-SEC2> ;
+.
+reprSciUnits:second
+  skos:exactMatch <http://qudt.org/vocab/unit/SEC> ;
+.
+reprSciUnits:secondSquared
+  skos:exactMatch <http://qudt.org/vocab/unit/SEC2> ;
+.
+reprSciUnits:siemens
+  skos:exactMatch <http://qudt.org/vocab/unit/S> ;
+.
+reprSciUnits:siemensPerMeter
+  skos:exactMatch <http://qudt.org/vocab/unit/S-PER-M> ;
+.
+reprSciUnits:steradian
+  skos:exactMatch <http://qudt.org/vocab/unit/SR> ;
+.
+reprSciUnits:tera
+  skos:exactMatch <http://qudt.org/2.1/vocab/prefix/Tera> ;
+.
+reprSciUnits:terahertz
+  skos:exactMatch <http://qudt.org/vocab/unit/TeraHZ> ;
+.
+reprSciUnits:tesla
+  skos:exactMatch <http://qudt.org/vocab/unit/T> ;
+.
+reprSciUnits:volt
+  skos:exactMatch <http://qudt.org/vocab/unit/V> ;
+.
+reprSciUnits:voltPerMeter
+  skos:exactMatch <http://qudt.org/vocab/unit/V-PER-M> ;
+.
+reprSciUnits:watt
+  skos:exactMatch <http://qudt.org/vocab/unit/W> ;
+.
+reprSciUnits:wattPerMeterPerKelvin
+  skos:exactMatch <http://qudt.org/vocab/unit/W-PER-M-K> ;
+.
+reprSciUnits:wattPerMeterSquared
+  skos:exactMatch <http://qudt.org/vocab/unit/W-PER-M2> ;
+.
+reprSciUnits:wattPerMeterSquaredPerKelvin
+  skos:exactMatch <http://qudt.org/vocab/unit/W-PER-M2-K> ;
+.
+reprSciUnits:wattPerMeterSquaredPerSteradian
+  skos:exactMatch <http://qudt.org/vocab/unit/W-PER-M2-SR> ;
+.
+reprSciUnits:weber
+  skos:exactMatch <http://qudt.org/vocab/unit/WB> ;
+.
+reprSciUnits:week
+  skos:exactMatch <http://qudt.org/vocab/unit/WK> ;
+.
+reprSciUnits:year
+  skos:closeMatch <http://qudt.org/vocab/unit/YR> ;
+  skos:closeMatch <http://qudt.org/vocab/unit/YR_Common> ;
+  skos:closeMatch <http://qudt.org/vocab/unit/YR_Sidereal> ;
+  skos:closeMatch <http://qudt.org/vocab/unit/YR_TROPICAL> ;
```

Comment:
Well, it's definitely picking nits, but I would suggest making YR_TROPICAL a closeMatch, and the other three a relatedMatch. Reason being that YR_TROPICAL is quite a bit closer numerically than the other 3, thus more likely to have been the intended meaning of the SWEET term for 'year'. The others are clearly different numbers.  (If only there had been a definition in SWEET...)

Reply:
Thanks John - like you say, if only we knew what SWEET intended

Reply:
If you can resolve and approve we can move forward now I think.

Reply:
(I accepted your suggestion and updated the code.)